test/python/qobj/test_qobj.py

```diff
@@ -153,7 +153,7 @@ def setUp(self):
                                    ],
                                    qubit_lo_freq=[4.9],
                                    meas_lo_freq=[6.9],
-                                   rep_time=1000),
+                                   rep_time=0.001),
```

Comment:
Why does this change?

Reply:
I changed it to fit the scaling from microseconds to seconds.(6 orders of magnitude) 
If this is not the best way to go about it I'm happy to change it back / find another solution.

Reply:
@lcapelluto Is this ok, or should it be changed back?

Reply:
Hey @eembees, sorry for such a late reply! I think for now it has to be changed back. It's a subtle point: the user facing side of Qiskit is being moved towards all basic SI units, but we are not yet altering the interface between Qiskit and the backends (the Qobj part of things). This makes a bit of extra work for us in the `assemble` part of Qiskit for the time being, but the Qobjs themselves will be unaltered.

Reply:
I'll leave a few comments around the PR to try to clarify